kola/tests/misc/nfs.go

```diff
@@ -123,15 +131,20 @@ systemd:
 		c.Fatal(err)
 	}
 
-	c.MustSSH(m2, fmt.Sprintf("stat /mnt/%s", path.Base(string(tmp))))
+	c.MustSSH(m2, fmt.Sprintf("stat /var/mnt/%s", path.Base(string(tmp))))
 }
 
 // Test that the kernel NFS server and client work within CoreOS.
 func NFSv3(c cluster.TestCluster) {
-	testNFS(c, 3)
+	testNFS(c, 3, "/tmp")
 }
 
 // Test that NFSv4 without security works on CoreOS.
 func NFSv4(c cluster.TestCluster) {
-	testNFS(c, 4)
+	testNFS(c, 4, "/tmp")
```

Comment:
Any reason not to just switch to `/`?

Reply:
I'll test again tomorrow morning but if I recall correctly that did not work on CL.

Reply:
Nah, I wouldn't spend time on it.